code/__HELPERS/roundend.dm

```diff
@@ -583,12 +583,39 @@
 		Trigger()
 		return
 
+///Returns a custom title for the roundend credit/report
+/proc/get_custom_title_from_id(datum/mind/mind, newline=FALSE)
+	if(!mind)
+		return
+
+	var/obj/item/card/id/I = mind.current?.get_idcard()
+	if(I)
+		if(I.registered_name == mind.name) // card must be yours
+			. = I.assignment // get the custom title
```

Comment:
Why are you assigning this to implicit return value if you are going to return explicitly?

Reply:
I felt like declaring a var was unnecessary, so I used dot as to build the return string instead of having a variable. Am I supposed to declare a var to build the return string?

Reply:
(And I am not sure if I understood your review correctly)

Reply:
Declare variable